lib/common/strings.c

```diff
@@ -894,27 +884,17 @@ pcmk__parse_ll_range(const char *srcstring, long long *start, long long *end)
 gboolean
 pcmk__str_in_list(GList *lst, const gchar *s, uint32_t flags)
```

Comment:
Not for this PR, but wouldn't s make sense as the first argument? Especially comparing it to any_of

Reply:
Yeah, that would make sense.  There aren't a ton of uses.  I could add it into this PR without too much work.